fiftyone/operators/delegated.py

```
@@ -454,8 +457,15 @@ def rerun_operation(self, doc_id):
         doc = self._repo.get(_id=doc_id)
         if doc is None:
             raise ValueError(f"No delegated operation with {doc_id=} found")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)